client/signup/steps/plans/index.jsx

```diff
@@ -138,6 +141,7 @@ module.exports = React.createClass( {
 		return (
 			<StepWrapper
 				flowName={ this.props.flowName }
+				goToNextStep={ isSkipPlansTestEnabled() ? this.onSelectPlan : null }
```

Comment:
+1 for setting this to `undefined` so the component can set a default if it has one.